src/index.html

```diff
@@ -52,9 +56,37 @@ <h2><span class="Datateknik">Datateknik</span> && <span class="InfoCom">Informat
       </p>
     </header>
     <div class="main-content"></div>
-    <div class="footer">
-      <button id="feedback-form">Rapportera problem med sidan</button>
-    </div>
+    <footer>
+      <div class="content">
+        <p>D-sektionen inom TLTH</p>
+        <p><a href="#">Kontakta oss</a></p>
+        <div class="address">
+          Besöksadress:<br>
+          E-huset<br>
+          Ole Römers väg 3<br>
+          22363 LUND
+        </div>
+        <div class="address">
+          Postadress:<br>
+          D-sektionen inom TLTH <br>
+          Box 118 <br>
+          22100 LUND
+        </div>
+        <p>Organisationsnummer: 845003-2878</p>
+        <button id="feedback-form">Rapportera problem med sidan</button>
+      </div>
+      <div class="divider"></div>
+      <div class="content">
+        <ul class="social-media">
```

Comment:
Do we want an Email button as well?

Reply:
Maybe, I was thinking that it will be part of the "Kontakt oss" section